Source/IntegrationTests/TestFiles/LitTests/LitTest/verification/proofDivision/isolatePaths.dfy

```diff
@@ -0,0 +1,49 @@
+// RUN: ! %verify --progress VerificationJobs --cores=1 %s &> %t
+// RUN: %OutputCheck --file-to-check "%t" "%s"
+// CHECK:Verified 1/4 of Assertion: assertion at line 28, through \[15,23\], could not prove all assertions
```

Comment:
See https://github.com/boogie-org/boogie/pull/970#discussion_r1813478602, I think we should label the numbers as line numbers in a path more clearly somehow.